pipelines/unified_quickstart/alt-2_calculate_avg_gpa_azure.wdl

```diff
@@ -0,0 +1,49 @@
+version 1.0
+
+workflow CalculateStudentGPA {
+    meta {
+        description: "Calculate the GPA (average) for a student."
+    }
+
+    input {
+        Float   math_score
+        Float   language_score
+        Float   science_score
+        Int     num_scores
```

Comment:
100% optional suggestion that I see users doing – make the image an input instead of hard-coding. Then you only need one copy of the WDL.
```suggestion
        Int     num_scores
        String  docker_image
```
You could even supply a default.
```suggestion
        Int     num_scores
        String  docker_image = "us.gcr.io/broad-dsp-gcr-public/base/python:3.12-debian"
```
This would require a corresponding `runtime` update:
```
runtime {
    docker: docker_image
}

Reply:
I'll create an improvement ticket for that. Other existing workflows in this repo will benefit too.